### DISCUSSION TITLE
特定のキーが動かない

### DISCUSSION THREAD

**Discussion by Kazuhiro-Mimaki** [Yowkees/keyball#463] [Q&A] [answered]
いくつか動作しないキーがあります。(以下の画像で青くなっていないキー)
ソケットの半田を見直したのですが、うまくいきません。左のキーボードについては横一列全てのキーが反応しないです。
よろしくお願いします。
<img width="1470" alt="スクリーンショット 2024-01-01 11 47 42" src="https://github.com/Yowkees/keyball/assets/62820515/89276bc9-30cb-4c40-9bbd-be1fdd4f424e">

| Left  | Right |
| ----- | ----- |
| ![IMG_8668](https://github.com/Yowkees/keyball/assets/62820515/158a371c-90ba-4032-a758-a84e80b989ec) | ![IMG_8667](https://github.com/Yowkees/keyball/assets/62820515/97ff701f-06e7-4e3e-b700-f374f7c7be04) |

**Comment by Kazuhiro-Mimaki** [comment, marked as answer]
左側のキーボードはpromicro裏、右側のキーボードはショートしていそうなダイオードのはんだごてをやり直すことで解決できました 🙏 
<img width="1470" alt="スクリーンショット 2024-01-01 17 06 53" src="https://github.com/Yowkees/keyball/assets/62820515/34519621-ad43-428b-a40b-6e51af4feeff">

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
返答が遅くなり申し訳ありませんでした。ご自身で半田付けを見直していただきありがとうございます。
無事に動くようになったようで安心しました。
ぜひ末永くご愛用いただければ幸いです。
